schemas/semconv-syntax.md

```diff
@@ -102,7 +105,12 @@ specificfields ::= spanfields
                |   metricfields
 
 spanfields ::= [events] [span_kind]
-eventfields ::= [name]
+
+eventfields ::= [name] [body]
```

Comment:
I believe name is required ([EBNF grammar](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form#:~:text=In%20computer%20science%2C%20extended%20Backus,as%20a%20computer%20programming%20language.)), so it should be

```suggestion
eventfields ::= name [body]
```

Reply:
Agree, I've applied this change locally to my PR but will wait to push any changes until after the `event_name` discussion (just to limit the number of changes / pushes)